.dockerignore

```diff
@@ -0,0 +1,5 @@
+Dockerfile
+.git
+.gitignore
+.env
+.env.template
```

Comment:
This list has the most important things, but some more examples of things you could ignore: `Vagrantfile`, `.vagrant`, `geckodriver.exe` (this one also be removed from git - it's in your gitignore but that doesn't affect files already being tracked. Remove it with `git rm --cached .geckodriver.exe`)